### PR TITLE
Add the real way to do it

### PR DESCRIPTION
**Pull request by wheresalice**
If this is meant to be a useful resource, then it should include the actual sensible way of doing it.

### CODE REVIEW

**Comment by rafalmaciejewski** [comment]
That's the boring way (or the noob way).

**Comment by hakluke** [comment]
Can't get this to work. Too complicated.

**Comment by chungy** [comment]
The Worst Way

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ Below are some simple methods for exiting vim.
 
 For real vim (and hacking) tips, follow [hakluke](https://twitter.com/hakluke) and [tomnomnom](https://twitter.com/tomnomnom) on twitter.
 
+## The real way
```
Comment (KabirKwatra) [inline README.md]:
```suggestion
## The incorrect way
```

**Pull request closed without merging**